cf-deployment.yml

```diff
@@ -1643,6 +1643,26 @@ instance_groups:
           reverse_log_proxy:
             cert: "((loggregator_tls_rlp.certificate))"
             key: "((loggregator_tls_rlp.private_key))"
+  - name: reverse_log_proxy_gateway
+    release: loggregator
+    properties:
+      http:
+        address: "0.0.0.0:8088"
+      logs_provider:
+        ca_cert: "((loggregator_rlp_gateway.ca))"
```

Comment:
We'd like to reference all CA Certs from their respective CA variables rather than certificate variables to enable easier certificate rotation in the future. So this line  will be `ca_cert: ((service_cf_internal_ca.certificate))`.